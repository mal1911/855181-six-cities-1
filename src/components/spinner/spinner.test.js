import React from "react";
import renderer from "react-test-renderer";
import Spinner from "./spinner";

it(`Spinner correctly renders`, () => {
  const tree = renderer
    .create(<Spinner/>).toJSON();
  expect(tree).toMatchSnapshot();
});
