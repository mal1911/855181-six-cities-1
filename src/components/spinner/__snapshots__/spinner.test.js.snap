// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner correctly renders 1`] = `
<div
  className="lds-css ng-scope"
  style={
    Object {
      "height": "200px",
      "width": "200px",
    }
  }
>
  <div
    className="lds-gear"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <div>
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
